app/database/admin/database_manager/index.ts

```diff
@@ -0,0 +1,365 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
```

Comment:
Can we keep the usage of `db` / `database` in naming constants / classes / types consistent? Maybe all interfaces / classes are `Database` and `db` is used for the constants or something along those lines? It's just making my eye twitch a bit 😅

Reply:
Done :)